site/_scss/blocks/_language-select.scss

```diff
@@ -0,0 +1,56 @@
+.language-select {
+
+  // Temporary class until web.dev gets rid of w- prefix.
+  .w-visually-hidden {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: auto;
+    margin: 0;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+  }
+
+  select {
+    /* stylelint-disable property-no-vendor-prefix */
+    -moz-appearance: none;
```

Comment:
Firefox supports the unprefixed `appearance` property since Firefox 80 (released in August 2020). Unless there's much traffic from Firefox <80, you can remove this line.

Reply:
Thanks @myakura !

Reply:
Thanks @myakura